mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -60,17 +62,18 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 	METHOD method_17903 asVoxelShape ()Lnet/minecraft/class_265;
 	METHOD method_17905 load (Lnet/minecraft/class_2784$class_5200;)V
 		ARG 1 levelProperties
+	METHOD method_27355 getProperties ()Lnet/minecraft/class_2784$class_5200;
```

Comment:
```suggestion
	METHOD method_27355 toConfig ()Lnet/minecraft/class_2784$class_5200;
```
converts this to a world border config; note this isn't getting because it is creating a new immutable config